src/content/docs/kubernetes-pixie/kubernetes-integration/installation/kubernetes-integration-install-configure.mdx

```diff
@@ -34,36 +34,252 @@ import kubernetesCke from 'images/kubernetes_logo_cke.png'
+
+[Learn more about custom app labels here.](https://docs.newrelic.com/docs/infrastructure/prometheus-integrations/install-configure-prometheus-agent/advanced-configuration/#enable-disable-integrations)
+
+The services you add to the submitted list will overwrite the data in `app_values` below and Prometheus will **only** scrape metrics from those services. 
+```
+  kubernetes:
+    # NewRelic provides a list of Dashboards, alerts and entities for several Services. The integrations_filter configuration
+    # allows to scrape only the targets having this experience out of the box.
+    # If integrations_filter is enabled, then the jobs scrape merely the targets having one of the specified labels matching
+    # one of the values of app_values.
+    # Under the hood, a relabel_configs with 'action=keep' are generated, consider it in case any custom extra_relabel_config is needed.
+    integrations_filter:
+      # -- enabling the integration filters, merely the targets having one of the specified labels matching
+      #    one of the values of app_values are scraped. Each job configuration can override this default.
+      enabled: true
+      # -- source_labels used to fetch label values in the relabel config added by the integration filters configuration
+      source_labels: ["app.kubernetes.io/name", "app.newrelic.io/name", "k8s-app"]
+      # -- app_values used to create the regex used in the relabel config added by the integration filters configuration.
+      # Note that a single regex will be created from this list, example: '.*(?i)(app1|app2|app3).*'
+      app_values: ["redis", "traefik", "calico", "nginx", "coredns", "kube-dns", "etcd", "cockroachdb", "velero", "harbor", "argocd"]
+```
+Above code snipped from: newrelic-prometheus-configurator/charts/newrelic-prometheus-agent/values.yaml
+
+##### Gather log data [#gather-logs]
+
+Users who wish to prioritize data ingest costs may choose to gather log data with minimal enrichment, which [drops Labels and Annotations](https://source.datanerd.us/fsi/k8s-config-generator/blob/master/deploy/helm/nri-bundle/charts/newrelic-logging/values.yaml#L106-L108) from their logs and only shares standard Kubernetes log data such as the [name of the cluster, container, namespace, and pod along with the message and timestamp](https://source.datanerd.us/fsi/k8s-config-generator/blob/master/deploy/helm/nri-bundle/charts/newrelic-logging/values.yaml#L116-L125).
+
+In minimal enrichment, only the following fields are retained:
+
+```
+Allowlist_key  container_name
+Allowlist_key  namespace_name
+Allowlist_key  pod_name
+Allowlist_key  stream
+Allowlist_key  log
+```
+
+Conversely, users who prefer more robust data can opt to fully enrich their logs by adding in label and annotation data to their log data. 
+
+**Full Enrichment Log Data** 
```

Comment:
Just a note to remove title case uses in this doc

Reply:
@zuluecho9 I had removed a lot of instances of title case locally. Please see the latest commit now that I've pushed that up.